driver-core/pom.xml

```diff
@@ -51,6 +51,12 @@
       <version>${metrics.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
This is to be able to build DAGs (directed acyclic graphs) for inter-UDT dependencies. See SchemaParser and UserType.